### PR TITLE
BUGFIX: Fix replacement of NodeTypeManager and add tests to cover this

### PR DESCRIPTION
**Pull request by dlubitz**
Fixes: #103

### CODE REVIEW

File: config/set/contentrepository-90.php [hunk 1]
```diff
@@ -435,6 +436,23 @@
...
+    // getNodeType(nodeTypeName: string): NodeType
+    // --> Compatible with 9.0
+    // getNodeTypes([includeAbstractNodeTypes: bool = true]): NodeType[]
+    // --> Compatible with 9.0
+    // getSubNodeTypes(superTypeName: string, [includeAbstractNodeTypes: bool = true]): NodeType[]
+    // --> Compatible with 9.0
+    // hasNodeType(nodeTypeName: string): bool
+    // --> Compatible with 9.0
+    // overrideNodeTypes(completeNodeTypeConfiguration: array): void
+    // --> Compatible with 9.0
```
Comment (mhsdesign) [inline config/set/contentrepository-90.php:454]:
```suggestion
    // overrideNodeTypes(completeNodeTypeConfiguration: array): void
    // --> Compatible with 9.0
```

this is not api and was also removed as far as i know

File: tests/Sets/ContentRepository90/Fixture/node-type-manager.php.inc [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+        // TODO 9.0 migration: !! NodeTypeManager::createNodeType() was never implemented and is removed in Neos 9.0.
+
+        $nodeTypeManager->createNodeType('Neos.MarketPlace:New');
+        return $node;
+    }
+
+    public function getNodeTypeManager(): \Neos\ContentRepository\Core\NodeType\NodeTypeManager
+    {
+        // TODO 9.0 migration: Make this code aware of multiple Content Repositories.
+        $contentRepository = $this->contentRepositoryRegistry->get(\Neos\ContentRepository\Core\SharedModel\ContentRepository\ContentRepositoryId::fromString('default'));
```
Comment (mhsdesign) [inline tests/Sets/ContentRepository90/Fixture/node-type-manager.php.inc:89]: i guess this is as good as it gets ... maybe we could write a rector that finds any $node instances in the current scope if possible and uses $node->contentRepositoryId then?

Comment (dlubitz) [reply tests/Sets/ContentRepository90/Fixture/node-type-manager.php.inc:89]: No, that's not easily possible

File: config/set/contentrepository-90.php (hunk 1 above)
Comment (dlubitz) [reply config/set/contentrepository-90.php:454]: but very fresh
https://github.com/neos/neos-development-collection/commit/24ffeff863eb202b3c7494641dbd73ddd74dfcb1

Comment (dlubitz) [reply config/set/contentrepository-90.php]: I've added a comment now. See [fc2a988](https://github.com/neos/rector/pull/121/commits/fc2a98863e8fdd8933d4d31632a30c29e5497df0)

**Pull request merged**